dashboard/test/models/user_test.rb

```diff
@@ -1014,273 +1014,6 @@ def assert_fails_email_uniqueness_validation(user)
-  end
-
-  test 'can get next_unpassed_progression_level, not tainted by other user progress' do
-    user = create :user
-    other_user = create :user
-    script = create(:script, :with_levels, levels_count: 5)
-    script.script_levels.each do |script_level|
-      UserLevel.create(
-        user: other_user,
-        level: script_level.level,
-        script: script,
-        attempts: 1,
-        best_result: Activity::MINIMUM_PASS_RESULT
-      )
-    end
-    assert_equal(1, user.next_unpassed_progression_level(script).chapter)
-  end
-
-  test 'can get next_unpassed_progression_level when most recent level is not passed' do
-    user = create :user
-    script = create(:script, :with_levels, levels_count: 5)
-
-    script.script_levels.each do |script_level|
-      next if script_level.chapter != 3
-      UserLevel.create(
-        user: user,
-        level: script_level.level,
-        script: script,
-        attempts: 1,
-        best_result: Activity::MINIMUM_FINISHED_RESULT
-      )
-    end
-
-    # The level we most recently had progress on we did not pass, so that's
-    # where we should go
-    assert_equal(3, user.next_unpassed_progression_level(script).chapter)
-  end
-
-  test 'can get next_unpassed_progression_level when most recent level is last level' do
-    user = create :user
-    script = create(:script, :with_levels, levels_count: 5)
-
-    script_level = script.script_levels.last
-    UserLevel.create(
-      user: user,
-      level: script_level.level,
-      script: script,
-      attempts: 1,
-      best_result: Activity::MINIMUM_PASS_RESULT
-    )
-
-    # User's most recent progress is on last level in script. There's nothing
-    # following it, so just return to the last level
-    assert_equal(script_level.chapter, user.next_unpassed_progression_level(script).chapter)
-  end
-
-  test 'can get next_unpassed_progression_level when most recent level is only followed by unplugged levels' do
-    user = create :user
-    script = create :script
-    lesson_group = create :lesson_group, script: script
-    lesson = create :lesson, script: script, lesson_group: lesson_group
-
-    script_levels = [
-      create(:script_level, script: script, lesson: lesson, levels: [create(:maze)]),
-      create(:script_level, script: script, lesson: lesson, levels: [create(:maze)]),
-      create(:script_level, script: script, lesson: lesson, levels: [create(:unplugged)]),
-    ]
-    create :user_script, user: user, script: script
-
-    UserLevel.create(
-      user: user,
-      level: script_levels[1].level,
-      script: script,
-      attempts: 1,
-      best_result: Activity::MINIMUM_PASS_RESULT
-    )
-
-    # User's most recent progress is on second last level of script, but none of
-    # the levels after it are "progression" levels. Just return to the last level
-    # we made progress on.
-    assert_equal(2, user.next_unpassed_progression_level(script).chapter)
-  end
```

Comment:
Did this test get dropped, or was it merged with another test?

Reply:
It is testing a similar scenario as in `'can get next_unpassed_progression_level if not completed any unplugged levels'`.